packages/core-data/src/resolvers.js

```diff
@@ -307,13 +307,25 @@ export const getEntityRecords =
 						response.headers.get( 'X-WP-TotalPages' )
 					);
 
+					if ( ! meta ) {
+						meta = {
+							totalItems: parseInt(
+								response.headers.get( 'X-WP-Total' )
+							),
+							totalPages: 1,
```

Comment:
Shouldn't this be using `X-WP-TotalPages`? We could use the `totalPages` constant from above:

```suggestion
							totalPages,
```

Reply:
The total pages definition above is used for the loop. The number of pages for unbound queries should be one, as it fetches all results and is paginated.

From previous work - #64772.

https://github.com/WordPress/gutenberg/blob/b5d6d7656f3861c6836cb12972bc724be4a2b1b2/packages/core-data/src/resolvers.js#L332-L335.

Reply:
Makes sense 👍